academy/2-cosmos-concepts/11-testing.md

```diff
@@ -51,52 +50,64 @@ Unit tests are your classic "smallest tests possible", and focus on a single mod
+Integration tests are one step wider in scope. They are still focused on your module but, instead of mocking the dependencies, now your test provisions a **minimal-viable application** that includes fully-fledged dependencies, including – crucially – those your own module needs.
 
 <HighlightBox type="info">
 
-A minimally-viable application is one that contains your module and all its dependencies, and their own continuing until all dependencies are satisfied, but nothing more.
+A minimal-viable application contains your module and all its dependencies, as well as their dependencies, but nothing more.
 
 </HighlightBox>
 
-In a well-designed testing environment, providing such fully-fledged dependencies should not be a concern of your module's tests. All the more so if your dependencies have dependencies of their own. You want to minimize such deep correlations between modules, even with regard to their tests.
+In a well-designed testing environment, providing such fully-fledged dependencies should not be a concern of your module's tests. All the more so if your dependencies have dependencies of their own. You want to minimize such deep correlations between modules, even in regard to their tests.
 
-This is why, with a view to minimizing correlations, from version 0.47 of the Cosmos SDK each module exposes functions to provision a minimally viable module. This way, your module only knows how to instantiate itself given fully-fledged dependencies, the _inputs_. An added benefit is that your module exposes explicitly the inputs it needs to instantiate.
+This is why, to minimize correlations, from version 0.47 of the Cosmos SDK on each module exposes functions to provide a minimal viable module. This way, your module only knows how to instantiate itself given fully-fledged dependencies, the _inputs_. An added benefit is that your module exposes explicitly the inputs it needs to instantiate.
 
-Your integration tests start by creating an app that instantiates the list of explicitly defined inputs required. For instance, when creating a minimally-viable app to integration-test the bank module no slashing module is provisioned, as slashing is a side concern instead of a dependency. Of course, each module requires a different minimally-viable app. To facilitate the creation of such an app and therefore of integration tests, the Cosmos team has also developed an in-house dependency injection.
+Your integration tests start by creating an app that instantiates the list of explicitly defined inputs required. For instance, when creating a minimal-viable app to integration-test the bank module, no slashing module is provisioned, as slashing is a side concern instead of a dependency. Of course, each module requires a different minimal-viable app. To facilitate the creation of such an app and therefore, of integration tests, the Cosmos SDK team has also developed an in-house dependency injection.
```

Comment:
```suggestion
Your integration tests start by creating an app that instantiates the list of explicitly defined inputs required. For instance, when creating a minimum-viable app to integration-test the bank module, no slashing module is provisioned, as slashing is a side concern instead of a dependency. Of course, each module requires a different minimum-viable app. To facilitate the creation of such an app and therefore, of integration tests, the Cosmos SDK team has also developed an in-house dependency injection.
```